src/r4300/new_dynarec/new_dynarec.c

```diff
@@ -10127,6 +10127,9 @@ int new_recompile_block(int addr)
   // to use, which can avoid a load-use penalty on certain CPUs.
   for(i=0;i<slen-1;i++)
   {
+    if(count_free_regs(regs[i].regmap)<=minimum_free_regs[i])
```

Comment:
I also suggest a comment: `// Avoid replacing an allocated temporary register`

Except if you think the code is explicit enough (I'm not an expert of dynarec).